package.json

```
@@ -138,6 +139,11 @@
 				"title": "Switch Neuro API user name",
 				"category": "NeuroPilot"
 			},
+			{
+				"command": "neuropilot.refreshExtensionDependencyState",
+				"title": "Get dependency extension states",
```

Title doesn't match what it's doing. Also mention the upcoming deprecation of this command.
```suggestion
				"title": "Refresh extension dependencies (workaround, will be deprecated soon)",
```